packages/mobile/src/app/ErrorMessages.ts

```diff
@@ -65,4 +62,7 @@ export enum ErrorMessages {
   ADDRESS_VALIDATION_PARTIAL_POORLY_FORMATTED = 'addressValidationPartialPoorlyFormatted',
   ADDRESS_VALIDATION_FULL_OWN_ADDRESS = 'addressValidationFullOwnAddress',
   ADDRESS_VALIDATION_PARTIAL_OWN_ADDRESS = 'addressValidationPartialOwnAddress',
+  PGPNP_QUOTA_ERROR = 'PGPNP_QUOTA_ERROR',
```

Comment:
Please restore these or the error messages won't work. Also I think their previous positions made more sense

Reply:
! woah
this was auto-formatted somehow